app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/asset/AssetMessageTypes.kt

```diff
@@ -64,68 +66,96 @@ internal fun MessageAsset(
             .clickable(if (isNotClickable(assetDownloadStatus, assetUploadStatus)) null else onAssetClick)
             .padding(dimensions().spacing8x)
     ) {
-        Column {
-            Text(
-                text = assetName,
-                style = MaterialTheme.wireTypography.body02,
-                maxLines = 2,
-                overflow = TextOverflow.Ellipsis,
-            )
-            ConstraintLayout(
-                Modifier
-                    .fillMaxWidth()
-                    .padding(top = dimensions().spacing8x)
-            ) {
-                val (icon, description, downloadStatus) = createRefs()
-                Image(
-                    modifier = Modifier
-                        .constrainAs(icon) {
-                            top.linkTo(parent.top)
-                            start.linkTo(parent.start)
-                            bottom.linkTo(parent.bottom)
-                        },
-                    painter = painterResource(R.drawable.ic_file),
-                    contentDescription = stringResource(R.string.content_description_image_message),
-                    colorFilter = ColorFilter.tint(MaterialTheme.wireColorScheme.badge)
-                )
+        if (assetUploadStatus == Message.UploadStatus.UPLOAD_IN_PROGRESS) {
+            // If asset is being uploaded, we don't want to show any asset metadata info
```

Comment:
Personal taste here, but I would prefer something like removing this comment and creating a val with this condition:

`val shouldHideAssetMetadata = assetUploadStatus == Message.UploadStatus.UPLOAD_IN_PROGRESS`

Nothing blocker anyway

Reply:
Not very opinionated about it, so why not?